src/NewField.cpp

```diff
@@ -100,6 +101,11 @@ static void apply_render_info_to_layer(Actor* layer,
 	}
 }
 
+bool operator<(NewField::field_draw_entry const& rhs, NewField::field_draw_entry const& lhs)
```

Comment:
Curious thought here. It's relatively simple to define `>`, `<=`, and `>=` with this setup. Should those operations be included?